src/github/github.ts

```diff
@@ -77,6 +79,10 @@ export class Github {
     this._token = opts.githubToken || Github.tokenFromFile('token');
     this._owner = opts.owner;
     this._repo = opts.repo;
+    this._proxy = opts.proxy;
+    // TODO: this is a temporary solution and needs to be fixed once
+    // https://github.com/mikedeboer/node-github/issues/539 gets fixed
+    this._proxy ? process.env.HTTPS_PROXY = this._proxy : null;
```

Comment:
with the changes above, you shouldn't need this anymore. If you still do, can you explain why / what you're trying to do here? (https://github.com/octokit/rest.js/issues/539 is fixed btw)

Reply:
It is still required. This change was not because of `cli`'s code but because of `github` module code. The problem was that the module did not had parameter `proxy` in it's `config` object because of which passing it as param to `GitHubApi` was not working. On checking the module's code i found that if `proxy` is not in `config` object they read it from `process.env`.
Even though the PR has been closed it looks like the project has also been migrated from `mikedeboer/node-github` to `octokit/rest.js`, which is not changed in `package.json`.